source/clear-linux/reference/compatible-hardware.rst

```diff
@@ -0,0 +1,113 @@
+.. _compatible-hardware:
+
+Compatible Hardware
+###################
+
+This document describes hardware that has been tested and confirmed as
+compatible with |CLOSIA|. This list is not comprehensive and will continue to
+grow.
+
+.. list-table:: **Table 1. Clear Linux Compatible Hardware**
```

Comment:
A suggestion:  Can we group these by architecture?  Also, I don't see the latest Xeon SKL-x on here (Purley)